unity-environment/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerContact.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
```

Comment:
Is it possible to use the generic ground contact script that is used for the humanoid walker ?

Reply:
the crawler needs to be able to detect when the target is touched. we'll probably need to detect even more types of objects going forward which is why i made a new script